Formula/libguess.rb

```diff
@@ -1,6 +1,6 @@
 class Libguess < Formula
   desc "Character set guessing library"
-  homepage "http://atheme.org/projects/libguess.html"
+  homepage "https://github.com/kaniini/libguess"
```

Comment:
This software is abandoned, not maintained, it has 2 installs in 90 days, let's remove the formula.